src/DotNetWorker/Invocation/DefaultFunctionExecutor.cs

```diff
@@ -19,7 +20,12 @@ public DefaultFunctionExecutor(ILogger<DefaultFunctionExecutor> logger)
 
         public async Task ExecuteAsync(FunctionContext context)
         {
-            var invoker = context.FunctionDefinition.Invoker;
+            if (!context.FunctionDefinition.Items.TryGetValue(DefaultFunctionDefinition.InvokerKey, out object? invokerObject) ||
+                invokerObject is not IFunctionInvoker invoker)
+            {
+                throw new InvalidOperationException("No function invoker found.");
```

Comment:
Should we be more detailed in this exception? Maybe describe the item/key expectation here? For more involved things we can also link to a doc (wiki/other) if needed.